src/state/products.js

```diff
@@ -39,6 +39,14 @@ export const extractedProductShape = pt.shape({
   date: pt.string.isRequired,
 });
 
+const DATA_URI_PATHS = [
```

Comment:
A comment or variable rename specifying that these are the allowed image mimetypes for data URIs would be neat.